File: Day93/DSA/Total_Waviness_of_Numbers_in_Range_I.java
Question:- https://leetcode.com/problems/total-waviness-of-numbers-in-range-i/

Code:- 

class Solution {
    public int totalWaviness(int num1, int num2) {

        int waviness = 0;

        for (int i = Math.max(100, num1); i <= num2; i++) {

            String s = String.valueOf(i);

            for (int j = 1; j < s.length() - 1; j++) {

                if ((s.charAt(j - 1) < s.charAt(j)) && (s.charAt(j) > s.charAt(j + 1)) ||
                        (s.charAt(j - 1) > s.charAt(j) && s.charAt(j) < s.charAt(j + 1))) {
                    waviness++;
                }
            }
        }
        return waviness;
    }
}

File: Day93/DSA/Total_Waviness_of_Numbers_in_Range_I.cpp
Question:- https://leetcode.com/problems/total-waviness-of-numbers-in-range-i/description/

Code:- 

class Solution {
public:
    int totalWaviness(int num1, int num2) {
        int waviness = 0;

        for (int i = max(100, num1); i <= num2; i++) {

            string s = to_string(i);

            for (int i = 1; i < s.size() - 1; i++) {

                if ((s[i - 1] < s[i] && s[i] > s[i + 1]) ||
                    (s[i - 1] > s[i] && s[i] < s[i + 1])){
                        waviness++;
                    }
            }
        }
        return waviness;
    }
};
